far/strmix.cpp

```diff
@@ -923,8 +923,8 @@ bool SearchAndReplaceString(
 	const auto Next = [&](size_t const Offset, size_t const Size)
```

Comment:
- Looks like we do not need `Size` here anymore, maybe remove it?
- There is [a very similar piece](https://github.com/FarGroup/FarManager/blob/a6c59d3c6af7ce340013c2449e047a487ca467c9/far/findfile.cpp#L1225-L1228) in findfile.cpp, it probably needs the same fix.

Reply:
- Doh!
- Fixed findfile.cpp